templates/_asset-card.html

```diff
@@ -0,0 +1,48 @@
+<div class="p-asset-card p-equal-height-row__col" style="">
+  <div class="p-equal-height-row__item" style="max-height:6rem;">
+    {% if asset.asset_type == "whitepaper" %}
+      <img class="p-asset-card--thumbnail" src="https://assets.ubuntu.com/v1/b061c401-White+paper.svg" alt="">
+    {% else %}
+      <img class="p-asset-card--thumbnail" src="https://assets.ubuntu.com/v1/fd84bbdc-Document-open.svg" alt="">
+    {% endif %}
+  </div>
+  <div class="p-equal-height-row__item">
+    {% if asset.name %}
+      <p><strong>{{ asset.name }}</strong></p>
+    {% else %}
+      <p><strong>{{ asset.file_path.split('.')[0] }}</strong></p>
+    {% endif %}
+    <p class="u-no-margin--bottom">File type: <strong>.{{ asset.file_path.split('.', 1)[1] }}</strong></p>
+    <p>Asset type: <strong>{{ asset.asset_type }}</strong></p>
+    <p>
+      Tags: 
+      {% for tag in asset.tags %}
+      <strong>{{ tag.name }}{% if not loop.last %},{% endif %}</strong>
+      {% endfor %}
+    </p>
+    <p>
```

Comment:
Just wondering if we could use a single template with some conditions for asset-card and asset-card-image

Reply:
I think we should keep them separate for now as we don't know how different they are going to look in the end.